ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/md_report.py

```diff
@@ -251,18 +251,18 @@ class AllTestResults(MDReportBase):
     """
 
     TABLE_HEADING: ClassVar[list[str]] = [
-        "| Test ID | Node | Test Categories | Test Description | Test | Test Result | Messages |",
-        "| ------- | ---- | --------------- | ---------------- | ---- | ----------- | -------- |",
+        "| ID | DUT | Categories | Test | Description | Inputs | Result | Messages |",
```

Comment:
Is "DUT" common terminology used by networking professionals? Should we spell out "Device Under Test" instead?

Reply:
We use DUT in the other sections of the report (carried over from the old jinja report) so I changed it here to be consistent but I agree I think we should go for Device Under Test everywhere.

Reply:
Or just `Device` or `Host` to keep it short?

Reply:
Fixed in 323d0618956f690a934d0a29de8304dc8bfec6bd